image/templates/helm/stackrox-secured-cluster/values-public.yaml.example

```diff
@@ -299,12 +299,10 @@
 ## Collector specific configuration.
 #collector:
 #
-#  # If enabled, the value provided in `collectionMethod` is applied to the collector configuration.
-#  # Otherwise, the default collection method is used.
+#  # Deprecated. This value has no effect.
 #  forceCollectionMethod: false
 #
 #  # Collection method to use when `forceCollectionMethod` is `true`. Can be one of:
```

Comment:
I guess we shouldn't be referencing this setting here, given that it is a no-op (and according to the `README.md` it has a default value of `false`?).

Reply:
Yeah good catch - I'll remove the reference (along with the variable itself as per your other comment)